docs/Project.toml

```
@@ -16,7 +16,7 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 [compat]
 Documenter = "0.24, 0.25, 0.26"
```

```suggestion
Documenter = "0.24, 0.25, 0.26, 0.27"
```
We could also add 0.27 here.

---

Sure.